test/suites/migration.sh

```diff
@@ -423,8 +429,14 @@ migration() {
   lxc_remote storage volume snapshot l1:"$remote_pool1" vol3
   lxc_remote storage volume set l1:"$remote_pool1" vol3 user.foo=postsnap1vol3
 
-  # check snapshot volumes and snapshots are refreshed
+  # check that a refresh doesn't change the volume's but snapshot's UUID.
```

Comment:
only snapshots that are changed right, can we add a check for that?

Reply:
aha! thanks, separate PR please for this, and a test if possible to avoid a regression in the future.

Reply:
Ok so that is partially fixed in https://github.com/canonical/lxd/pull/12893. I found that the test isn't checking the right thing here either, so probably it's now a combination of things I need to check. I'll post here again when it's ready.

Reply:
The remainder is fixed in https://github.com/canonical/lxd/pull/12894.

Reply:
The tests now reflect the correct behavior of not refreshing the snapshot and its UUID.